packages/rest/README.md

```diff
@@ -52,16 +52,35 @@ Application options.
 const app = new RestApplication({
   rest: {
     port: 3001,
+
+    // Optional configuration for API explorer and spec
+    apiExplorer: {
+      // URL for the hosted API Explorer UI
+      // default to https://loopback.io/api-explorer
+      explorerUrl: 'http://petstore.swagger.io',
+      // Set `servers` based on HTTP request headers, default to `false`
+      setServersFromRequest: false,
+      // Template the OpenAPI spec, which will be filled with `paths` from
+      // controllers.
+      openApiSpec: {
```

Comment:
I think the base template for OpenAPI spec should not be set in the `apiExplorer` object. It should be it's own top level property under `rest`.

Reply:
Good point. Fixed.